### PR TITLE
fixes everyone being literally retarded

### LINKED ISSUE #33675: Everyone is stupid
Round ID 80437
Everyone has brain damage and is headbutting doors and has a stupid expression on their face.
We know the game should be realistic but this is kinda overkilll.

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #33675
broken off from #33663

### CODE REVIEW

**Comment by AnturK** [comment]
Wouldn't it be easier to allow this format ? Seems like common usage.

**Comment by ShizCalev** [comment]
Actually it was incorrect in the source PR. Fixed.

**Comment by XDTM** [comment]
There's also an unreported issue with remove_disability, the second argument needs to be a list instead of text. You might want to pull those changes too.

File: code/modules/mob/living/status_procs.dm [hunk 1]
```diff
@@ -146,10 +146,12 @@
 	else
 		disabilities[disability] |= list(source)
 
-/mob/living/proc/remove_disability(disability, list/sources)
+/mob/living/proc/remove_disability(disability, sources)
 	if(!disabilities[disability])
 		return
 
+	sources = list(sources)
```
Comment (Cyberboss) [inline code/modules/mob/living/status_procs.dm]: `if(!islist(sources)`?

Comment (Cyberboss) [reply code/modules/mob/living/status_procs.dm]: @XDTM ?

Comment (XDTM) [reply code/modules/mob/living/status_procs.dm]: If the list() proc does not do that already, then yes

**Pull request merged**